### ISSUE TITLE
EntityRenderer Crash v12

### ISSUE THREAD

**Issue by Shahelion** [ReikaKalseki/Reika_Mods_Issues#704]
None of your mods seem to have crashed themselves, but it was triggered when I updated yours.

If the cause is something else, then I shall be happy to direct this to them.

https://gist.github.com/Zsashas/9b16124799a37cded6ee3d0ae4a3ca22

**Comment by Aivech** [comment]
What happens when you remove Witchery?

**Comment by Shahelion** [comment]
Looks like it may be caused by the Betweenlands mod...will re-open if this is not the case.

**Comment by ReikaKalseki** [comment]
This looks like an ASM conflict with my new EntityRenderEvent. Does Witchery have ASM? Does Betweenlands?

**Comment by kolatra** [comment]
BetweenLands does yes, you can look at it here. https://github.com/Angry-Pixel/The-Betweenlands/tree/master/java/thebetweenlands/core

Not sure about Witchery because it's closed source.

**Comment by ReikaKalseki** [comment]
And he has an ASM hook into RenderManager, though it does not look incompatible at first glance. @Angry-Pixel: Do you have more information?

**Comment by FnordMan** [comment]
I'm also getting crashes with v12b, only with fakeplayer. Attached the fml client log for the crash. Also attached a modlist

[modlist.txt](https://github.com/ReikaKalseki/Reika_Mods_Issues/files/194564/modlist.txt)

[fml-client-latest.log.txt](https://github.com/ReikaKalseki/Reika_Mods_Issues/files/194561/fml-client-latest.log.txt)

**Comment by ReikaKalseki** [comment]
Link just the crashlog.

**Comment by FnordMan** [comment]
There isn't any crashlog for some reason, that's why I included the forge log. The crash is at the very bottom of the file.

**Comment by Shahelion** [comment]
That looks like a different error, please open a separate issue.

**Comment by ReikaKalseki** [comment]
Looking more closely at it, there is an ASM conflict into EntityRenderer. The only new hook I have for that is the getMouseoverEvent, into `func_78473_a`/`getMouseOver` which he also seems to ASM into.

The good news is that it looks like he is adding a hook. If he posts it as a Forge event - or uses mine - the two can be compatible.

https://github.com/Angry-Pixel/The-Betweenlands/issues/251

**Comment by pau101** [comment]
Originally I wrote The Betweenlands ASM for EntityRenderer#getMouseOver to scrap the existing code and just reroute it to my own function that included custom event hooks, but I've fixed that in Angry-Pixel/The-Betweenlands@4f7bf02 so the ASM is not far less likely to cause incompatibilities and doesn't cause this crash.
Version 1.0.1 of The Betweenlands will be released after a few more days for any other bugs that come in to be resolved, so for the time being The Betweenlands 1.0.0 is incompatible with Dragon API. I believe this issue can be closed.
